tests/braille-specs/ipa-unicode-range.yaml

```diff
@@ -0,0 +1,183 @@
+# LibLouis: International Phonetic Alphabet
+#
+# Copyright (C) 2019 Ludovic Oger <oger.ludovic@gmail.com>
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+
+display: unicode-without-blank.dis
+table: fr-bfu-comp8.utb
+table: IPA.utb
```

Comment:
I'd also add `table: ipa-unicode-range.uti`.

Reply:
OK, did that